### ISSUE TITLE
Suddenly stopped working

### ISSUE THREAD

**Issue by doronbehar** [raghur/vim-ghost#33]
I can't figure out why the extension stopped working lately, I've doubled checked `g:ghost_port` and it's the same as the one configured for the extension. The logs don't show anything interesting:
```
2019-04-12 19:15:31,677 [INFO @ session.py:request:101] 32233 - 'Received error: [1, b'Key not found: ghost_cmd']
2019-04-12 19:15:31,680 [INFO @ session.py:request:101] 32233 - 'Received error: [1, b'Key not found: ghost_darwin_app']
2019-04-12 19:15:32,183 [INFO @ ghost.py:service_actions:105] 32233 - server started
```

I'm not sure whether it's a problem with vim-ghost or GhostText the extension. It seems there is not even a connection establishment so it might be something wrong with my setup. But, I've tested it with no other plugins installed with `env XDG_CONFIG_HOME=/some/other/dir nvim +GhostStart` and it didn't help to get some clues.

I really miss using vim-ghost so I kindly ask for help.

**Comment by raghur** [comment]
Is your `pynvim`/`neovim` python client plugin up to date? That might be one thing to check if you're on nvim nightly
builds.  The first log line is a bit weird since the code has an explicit check to see if the var is defined before
trying to access it.
Do you have anything in `:messages`?
What happens when you click the ghost button on the browser? No errors? There should be something in the log if the
browser plugin makes a request to the http server.

-R

**Comment by doronbehar** [comment]
`:messages` doesn't show anything interesting besides the `Ghost: server started on port 4465`. I'm using `pynvim` `0.3.2`:
```
$ pip show pynvim
Name: pynvim
Version: 0.3.2
Summary: Python client to neovim
Home-page: http://github.com/neovim/python-client
Author: Thiago de Arruda
Author-email: tpadilha84@gmail.com
License: Apache
Location: /usr/lib/python3.7/site-packages
Requires: msgpack, greenlet
Required-by: neovim-remote
```
My Neovim version is `0.3.4` on Arch Linux. I've checked with `netstat -lntup` that a program actually opens port `4465`:
```
tcp        0      0 127.0.0.1:4465          0.0.0.0:*               LISTEN      14093/python3       
```

When I click on the ghost button on the browser, I don't see any errors and I can see the blue padding around the text box. Should I attempt at debugging it? I can send debug messages of `nvim -Vdebug.log +GhostStart` but I'm not sure what debug level you would wish to get.

**Comment by raghur** [comment]
Ok.. can you confirm that the browser add-on is also using 4465? Default is 4001afaicr. Browser pointing to different Port and plugin on a different Port would explain lack of any errors

**Comment by doronbehar** [comment]
Of course, I've doubled checked that a hundred times if not more. I've tried even setting it to something else (8822) and no luck. Port 4001 is taken on my machine. After I've disabled the service that's running on port 4001 on and I tried using the extension and the plugin with the default settings that finally worked! The problem is that either the extension or vim-ghost don't respect this setting.
